backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -410,6 +410,14 @@ class UserDigestSchema(marshmallow.Schema):
     workspace_ids = marshmallow.fields.List(marshmallow.fields.Int(example=3))
 
 
+class KnownMembersUserDigestSchema(UserDigestSchema):
+    """
+    UserDigestSchema with an "is_active" boolean added
+    """
+
+    is_active = marshmallow.fields.Bool()
```

Comment:
Why not complete the UserDigestSchema with the `is_active` field? This would lead to less code to maintain (so less bug to fix)... or there is a good reason to split into 2 models?

Reply:
@Millefeuille42

Reply:
This is to prevent modifying routes that use this Schema. Modifying it would either:
- Need to edit all the routes using this Schema to provide the value for `is_active`
- Make it optional and have inconsistent results between all routes using this endpoint

While this generates more "noise", this permits to 
1. Contain the changes in the scope of the issue (do not engage in a refactor)
2. Have consistent, clear and defined response schemas on endpoints (and thus in the swagger docs)

Reply:
I'm assuming this would make every endpoint already using the UserDigestSchema to now return a new property "is_active: true" which would not be used
I don't know if it's an issue but we might expect changes in unit tests and api documentation

Reply:
> Require us to edit all the routes using this Schema to provide the value for is_active

This is straight without any dev work, I think.

> Require us to make it optional and have inconsistent results between all routes using this endpoint

Why do you want to make it optional ?

> using the UserDigestSchema to now return a new property "is_active: true" which would not be used

Of course this is not used (it is not available;)

Doing like this increase complexity. I can understand "you dont want to break anything; this is the reason for unit tests existance".

Reply:
I understand the point "do not break existing code". I'm pretty confident we can do it without to break anything. @Millefeuille42 let's try to add the property to UserDigestSchema and we talk about what it breaks

Reply:
done